### DISCUSSION TITLE
Error retrieving draft pull request via GraphQL

### DISCUSSION THREAD

**Discussion by BasThomas** [community/community#24464] [Apps, API and Webhooks]
<p>reference D2E9:0A99:11B1BA0:2AD4171:5C6A8C98.</p>
<p>I’m assuming this has something to do with the fact this is a draft pull request — any pointers?</p>

**Comment by nickvanw** [comment]
<p>Hey <a class="mention" href="/basthomas">@basthomas</a>!</p>
<p>This is a bug related to draft PRs indeed! We’re working on a fix right now that should be deployed very soon! I’ll update this issue when that’s gone through.</p>

**Comment by BasThomas** [comment]
<p>Seems like it has been fixed!</p>

**Comment by nickvanw** [comment, marked as answer]
<p>Hey,</p>
<p>Yep! The fix went out after I left for the day yesterday, it looks like you beat me to it on the thread.</p>
